src/main/scala/au/com/agiledigital/toolform/app/ToolFormApp.scala

```diff
@@ -64,7 +63,10 @@ object ToolFormApp extends App {
           } text "the path to output the generated file/s",
           opt[Unit]('d', "generate-docker-compose")
             .action((_, c) => c.copy(generateTaskConfiguration = c.generateTaskConfiguration.copy(generateTaskOutputType = GenerateTaskOutputType.DockerComposeV3)))
-            .text("generate a Docker Compose v3 file as output (default)")
+            .text("generate a Docker Compose v3 file as output (default)"),
+          opt[Unit]('k', "generate-kubernetes")
```

Comment:
Will need to be converted to decline sub-commands. Expect invocation to look like:

`toolform generate minikube -i ... -o ...`

`toolform generate --help` should like the available generators. This should work if decline is correctly configured.

@timechanter are you happy with that output?